### PR TITLE
KE2: Extract parenthesized expressions

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

File: java/kotlin-extractor2/src/main/kotlin/entities/MethodCall.kt [hunk 1]
```diff
@@ -32,7 +33,11 @@ fun KotlinFileExtractor.extractMethodCall(
     val args = call.valueArguments
         .map { arg ->
             val expr = arg.getArgumentExpression()
-            val p = argMapping[expr]
+
+            // `argMapping` seems to drill into parenthesized expressions
```
Comment (igfoo) [inline java/kotlin-extractor2/src/main/kotlin/entities/MethodCall.kt:37]: Do you know where this happens?

Comment (tamasvajk) [reply java/kotlin-extractor2/src/main/kotlin/entities/MethodCall.kt:37]: I don't, but I found the utility function `KtPsiUtil.deparenthesize` which does something similar but also handles `KtAnnotatedExpression` and `KtLabeledExpression`. So we might also need to cover these too.

Comment (igfoo) [reply java/kotlin-extractor2/src/main/kotlin/entities/MethodCall.kt:37]: Let's add a TODO comment so we ensure that we handle all cases properly

**Review by igfoo — APPROVED** [review]
Otherwise LGTM.

**Pull request merged**